### PR TITLE
adding sixtools recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: https://github.com/nsls-ii-forge/sixtools-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
ATTN @mrakitin once [rixs PR](https://github.com/conda-forge/staged-recipes/pull/15670) is merged, it's still waiting on you agreement to be maintainer, I will retrigger the checks.

File: recipes/sixtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "sixtools" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (mrakitin) [inline recipes/sixtools/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/sixtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (mrakitin) [inline recipes/sixtools/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/sixtools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  sha256: 38950e920b9ae28979f1dee66756763714b601ed1553ec864a8278de6c295105
+
+build:
+  number: 0
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (mrakitin) [inline recipes/sixtools/meta.yaml:15]:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/sixtools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - databroker
+    - ipywidgets
+    - matplotlib
```
Comment (mrakitin) [inline recipes/sixtools/meta.yaml:25]:
```suggestion
    - matplotlib-base
```

File: recipes/sixtools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    - pytest
+  commands:
+    - pytest --pyargs sixtools.tests
+
+about:
+  home: "https://github.com/NSLS-II-SIX/sixtools"
+  license: BSD-3-Clause
+  license_family: "BSD"
+  license_file: "LICENSE"
+  summary: "Software for performing resonant inelastic xray scattering analysis at NSLS-II"
```
Comment (mrakitin) [inline recipes/sixtools/meta.yaml:45]:
```suggestion
  home: https://github.com/NSLS-II-SIX/sixtools
  license: BSD-3-Clause
  license_family: BSD
  license_file: LICENSE
  summary: Software for performing resonant inelastic xray scattering analysis at NSLS-II
```

**Review by mrakitin — APPROVED** [review]
A couple of formatting/styling suggestions.

@conda-forge/staged-recipes, I agree to be a maintainer.

**Comment by mrakitin** [comment]
Need https://github.com/conda-forge/staged-recipes/pull/15670 to be merged first, then this PR can be "power-cycled" to trigger a new build with the released package.

Alternatively, this recipe can be added to the `rixs` PR, so they both come together (and this one can be simply closed without merge.)

@dopplershift, what route would you prefer? Thank you!

**Comment by mrakitin** [comment]
We decided to have a combined PR with both of those recipes.

**Pull request closed without merging**